Camera2SlowMotion/app/src/main/java/com/example/android/camera2/slowmo/MainActivity.java

```diff
@@ -38,7 +38,7 @@
 public class MainActivity extends AppCompatActivity {
     private Integer totalFrames=0;
```

Comment:
I think these two should be final and CONSTANT_CASE two right?

Reply:
`safeFrames` can be constant but I'm setting `totalFrames` from metadata later on. Initialized here as I didn't wanted a garbage value to indicate that `totalFrames` is set.